style.css

```diff
@@ -0,0 +1,63 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  background-image: url(/assets/image_geometry_1.png);
```

Comment:
- Again here you are trying to add the background images, but they are also not showing on your portfolio. Please fix this issue 📝.
- **Here is how it looks now:** 
![image](https://user-images.githubusercontent.com/43172164/198092684-28d191f8-ec48-4a31-bf0d-f2bb083c41fa.png)
- **And this is how it should look:** 
![image](https://user-images.githubusercontent.com/43172164/198092850-24607b8d-d85c-4adf-a2fa-0004fb385065.png)